app/models/rental.py

```diff
@@ -0,0 +1,19 @@
+from flask import current_app
+from app import db
+from app.models.customer import Customer
+from app.models.video import Video
+from datetime import datetime
+
+class Rental(db.Model):
+    rental_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    customer_id = db.Column(db.Integer, db.ForeignKey('customer.id'), primary_key=True)
+    video_id = db.Column(db.Integer, db.ForeignKey('video.id'), primary_key=True)
```

Comment:
Neither video nor customer have a property id.  This was causing the migration errors.
```suggestion
    customer_id = db.Column(db.Integer, db.ForeignKey('customer.customer_id'), primary_key=True)
    video_id = db.Column(db.Integer, db.ForeignKey('video.video_id'), primary_key=True)
```